tests/descriptor_tests/test_api.py

```diff
@@ -161,6 +163,7 @@ def test_alt_cache_root(self):
             "app_store",
             "tk-testaltcacheroot2",
             "v0.4.3")
+        self.addCleanup(shutil.rmtree, app_root_path)
```

Comment:
`app_root_path` is part of the `bundle_root` - why does it need cleaning up? Is it not handled as part of deleting `bundle_root`?